proto/osmosis/cosmwasmpool/v1beta1/model/pool.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package osmosis.cosmwasmpool.v1beta1;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v15/x/cosmwasmpool/model";
+
+message CosmWasmPool {
+  option (gogoproto.goproto_getters) = false;
+  option (gogoproto.goproto_stringer) = false;
+  option (cosmos_proto.implements_interface) = "PoolI";
+  string pool_address = 1 [ (gogoproto.moretags) = "yaml:\"pool_address\"" ];
+  string contract_address = 2
+      [ (gogoproto.moretags) = "yaml:\"contract_address\"" ];
+  uint64 pool_id = 3;
+  uint64 code_id = 4;
+  bytes instantiate_msg = 5
```

Comment:
similarly, the instantiate message can be retrieved from the contract history, but that's a bit harder so might be useful here to determine something about the pool

Reply:
going to remove for now - I agree that this is likely redundant